base_location_nuts/wizard/nuts_import.py

```diff
@@ -175,7 +175,7 @@ def create_or_update_nuts(self, node):
         nuts = nuts_model.search([('level', '=', data['level']),
                                   ('code', '=', data['code'])])
         if nuts:
-            nuts.write(data)
+            nuts.filtered(lambda n: n.not_updatable is False).write(data)
```

Comment:
You can do ```lambda n: not n.not_updatable``` but it does the same so it's not really important.

Reply:
yes @tarteo but I wanted to avoid the double negation

Reply:
I prefer the other option too. Double negation is correct, and more in programming

Reply:
ok this is a plebiscite then, fixed that in https://github.com/OCA/partner-contact/pull/628/commits/1d555b8d43169161d4f3f80c0e770894dbdb87e9